### PR TITLE
Completed TC04 31/10/2021

### PR DESCRIPTION
**Pull request by HoanVanHuynh**

### CODE REVIEW

File: src/main/java/page_objects/LoginPage.java [hunk 1]
```diff
@@ -50,4 +55,14 @@ public String getWelcomeMessage() {
...
+    public String getTopicContentAtTop() {
+        return this.getLblTopicContentAtTop().getText();
+    }
 }
+
+
+
+
+
+
```
Comment (anhlvt) [inline src/main/java/page_objects/LoginPage.java]: redundant

File: src/main/java/page_objects/LoginPage.java [hunk 2]
```diff
@@ -50,4 +55,14 @@ public String getWelcomeMessage() {
     public String getErrorMessageAtTop() {
         return this.getLblErrorMessageAtTop().getText();
     }
+
+    public String getTopicContentAtTop() {
+        return this.getLblTopicContentAtTop().getText();
+    }
 }
+
```
Comment (tanmle) [inline src/main/java/page_objects/LoginPage.java]: why do we need too many downlines here?

File: src/main/java/page_objects/LoginPage.java (hunk 1 above)
Comment (HoanVanHuynh) [reply src/main/java/page_objects/LoginPage.java]: thank you so much, I'll delete that immediately

**Review by anhlvt — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

File: src/main/java/page_objects/LoginPage.java (hunk 2 above)
Comment (HoanVanHuynh) [reply src/main/java/page_objects/LoginPage.java]: I have already deleted redundant lines from Mr. Anh reviewer

**Pull request merged**